src/rules/enforce-boolean-naming-prefixes.ts

```
@@ -304,8 +357,16 @@ export const enforceBooleanNamingPrefixes = createRule<Options, MessageIds>({
             const calleeName = leftSide.callee.name;
-            return approvedPrefixes.some((prefix) =>
-              calleeName.toLowerCase().startsWith(prefix.toLowerCase()),
+            const lowerCallee = calleeName.toLowerCase();
+
+            // For assert*-style utilities, only treat as boolean if we can confirm boolean return type
+            if (lowerCallee.startsWith('assert')) {
+              return identifierReturnsBoolean(calleeName);
+            }
+
+            return approvedPrefixes.some(
+              (prefix) => prefix !== 'asserts' &&
```

ESLint

[error] Insert `⏎···············`

(prettier/prettier)